networkx/algorithms/centrality/tests/test_betweenness_centrality.py

```diff
@@ -655,3 +696,80 @@ def test_normalized_weighted_graph(self):
         norm = len(G) * (len(G) - 1) / 2
         for n in sorted(G.edges()):
             assert b[n] == pytest.approx(b_answer[n] / norm, abs=1e-7)
+
+    def test_weighted_multigraph(self):
+        """Edge betweenness centrality: weighted multigraph"""
+        eList = [
+            (0, 1, 5),
+            (0, 2, 4),
+            (0, 3, 3),
+            (0, 3, 3),
+            (0, 4, 2),
```

Comment:
This test uses a graph with identical multiedge from `(0, 3)` but those two edges tie with each other and no other shortest path.  Is there a way to test a multiedge that ties not only with its parallel twin, but also with a path that doesn't traverse this node-pair?  Otherwise we never fully test the correct splitting of ties.  I'm getting muddled trying to come up with an example, but is this testing idea clear?

Reply:
Adding ```(0, 1, 4)``` should do, as currently the shortest weighted path from 0 to 1 is of length 4 via path ```[0,3,1]```.
Then we have two paths from 0 to 1 using different edges. This is the request, right?